CRM/Core/BAO/UFGroup.php

```diff
@@ -1459,22 +1459,26 @@ public static function add(&$params, $ids = []) {
     if (!empty($params['group_type']) && is_array($params['group_type'])) {
       $params['group_type'] = implode(',', $params['group_type']);
     }
+
+    $hook = empty($params['id']) ? 'create' : 'edit';
+    CRM_Utils_Hook::pre($hook, 'UFGroup', ($params['id'] ?? NULL), $params);
+
     $ufGroup = new CRM_Core_DAO_UFGroup();
     $ufGroup->copyValues($params);
 
-    $ufGroupID = CRM_Utils_Array::value('ufgroup', $ids, CRM_Utils_Array::value('id', $params));
-    if (!$ufGroupID && empty($params['name'])) {
+    if (empty($params['name'])) {
```

Comment:
I don't think we can drop ufGroupID from this check. I think you removed it in order to use the preferred shorter syntax for the hook - but it looks like the variable is still needed